tutorials/terraform/data-transfer/data-transfer-mmy-ydb.tf

```diff
@@ -0,0 +1,117 @@
+# Infrastructure for the Yandex Cloud Managed Service for MySQL cluster, Managed Service for YDB and Data Transfer.
```

Comment:
Infrastructure for the Yandex Cloud Managed Service for MySQL cluster, Managed Service for YDB, and Data Transfer.